nvflare/edge/assessors/model_update.py

```diff
@@ -11,41 +11,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import random
+
 import threading
 import time
 from typing import Optional
 
-import numpy as np
-
-from nvflare.apis.dxo import DXO, DataKind
 from nvflare.apis.event_type import EventType
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.shareable import Shareable
 from nvflare.app_common.abstract.learnable_persistor import LearnablePersistor
-from nvflare.app_common.abstract.model import ModelLearnable, make_model_learnable, model_learnable_to_dxo
-from nvflare.app_common.app_constant import AppConstants
+from nvflare.app_common.abstract.model import ModelLearnable, model_learnable_to_dxo
 from nvflare.app_common.app_event_type import AppEventType
-from nvflare.edge.aggregators.model_update_dxo import ModelUpdateDXOAggregator
 from nvflare.edge.assessor import Assessment, Assessor
-from nvflare.edge.mud import BaseState, ModelUpdate, StateUpdateReply, StateUpdateReport
-
-
-class _ModelState:
-
-    def __init__(self, aggr: ModelUpdateDXOAggregator):
-        self.aggregator = aggr
-        self.devices = {}
-        self.last_update_time = None
-
-    def accept(self, model_update: ModelUpdate, fl_ctx: FLContext):
-        self.last_update_time = time.time()
-        self.devices.update(model_update.devices)
-        return self.aggregator.accept(model_update.update, fl_ctx)
+from nvflare.edge.assessors.device_manager import DeviceManager
+from nvflare.edge.assessors.model_manager import ModelManager
+from nvflare.edge.mud import BaseState, StateUpdateReply, StateUpdateReport
 
 
 class ModelUpdateAssessor(Assessor):
```

Comment:
It should have args for "model_manager_component_id" and "device_manager_component_id" to allow different implementations to be used.